local_settings.py.example

```diff
@@ -8,6 +8,15 @@ ALLOWED_HOSTS = [
         "relate.example.com",
         ]
 
+# Comment or leave the following line as is if you can run the
+# following command:
+#        python manage.py migrate sites
+# If that works without problem, edit in admin the "sites" block,
+# update the first/only record to your domain.
+# Otherwise, replace the following string with your own domain.
+
+RELATE_SITE_DOMAIN = "www.example.com"
+
```

Comment:
Sorry for my bad expression. My idea is that,  if sites can be properly migrated, then use ``Site_ID=1`` or something to generate the full url. 

Anyway, I've abandon that idea, use BASE_URL instead.